scripts/stack/docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+
+networks:
+  connectors_stack_net:
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTICSEARCH_VERSION}
+    environment:
+      - "discovery.type=single-node"
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
I think 512m is too small for us - we often have problems even with 1GB deployment. Can we make it 2GB here?

Reply:
Thanks for the tip. Will do. I'll also update the README to reflect these constraints too.